pyrdp/mitm/DeviceRedirectionMITM.py

```diff
@@ -270,6 +276,22 @@ def handleCloseResponse(self, request: DeviceCloseRequestPDU, _: DeviceCloseResp
             self.saveMapping()
 
 
+    def handleClientLogin(self):
+        """
+        Handle events that should be triggered when a client logs in.
+        """
+
+        if self.state.candidate or self.state.inputBuffer:
+            self.log.info("Credentials candidate: %(candidate)s", {"candidate" : (self.state.candidate or self.state.inputBuffer) })
```

Comment:
Mention "heuristic" in the log message and for the json output "candidate" is too broad of a term. Use "candidate_credentials" instead.